smarts/env/hiway_env.py

```diff
@@ -81,6 +83,7 @@ def __init__(
         envision_endpoint=None,
         envision_record_data_replay_path=None,
         zoo_workers=None,
+        authkey=b"secret",
```

Comment:
```suggestion
        auth_key=None,
```

Reply:
Changed default value of auth_key of type byte string to be b"None".